### PR TITLE
Set the input tag for the PileupSummaryInfo collection retrieval

### PR DESCRIPTION
**Pull request by forthommel**
Added the capability to set the input tag for the PileupSummaryInfo collection retrieval. This new method can be used for the interaction with MiniAOD files (using slimmedAddPileupInfo instead of addPileupInfo as a label for the vector&lt;PileupSummaryInfo&gt; collection)

### CODE REVIEW

**Comment by mdhildreth** [comment]
please test

**Comment by forthommel** [comment]
Many thanks for digging up this rather old PR! Is there any plan to have this integrated in a future release?

**Comment by davidlange6** [comment]
@forthommel - indeed, seems we lost it

naively it seems more natural to add your new InputTag as an optional argument to the LumiReWeighting constructors as done for other parameters. Did you consider that?

**Comment by forthommel** [comment]
Sure! but for historical reasons I did not do it... :)  do you think it would be preferable to do so?

**Comment by davidlange6** [comment]
yes - otherwise is isn't much reason to implement the PR that I see(??) @forthommel thx

**Comment by forthommel** [comment]
Hi! Is this any better?

**Comment by forthommel** [comment]
Hi @davidlange6, all,
Should I take any actions for this PR to proceed? Or close it...?

**Comment by monttj** [comment]
@cmsbuild please test

**Comment by forthommel** [comment]
Hi @monttj ! Could you let me know if the tests were indeed started?

**Comment by monttj** [comment]
@cmsbuild please test
It is strange. Let me try again.

**Comment by monttj** [comment]
@cmsbuild please test

**Comment by forthommel** [comment]
Great to see it passed the CI tests. May I backport it to 8_0_X?

**Comment by monttj** [comment]
+1
@forthommel 
Yes, could you backport it to 8_0_X?

**Comment by davidlange6** [comment]
+1
